### PR TITLE
core: add eos-kolibri

### PR DESCRIPTION
**Pull request by erikos**
We finally integrate Kolibri system wide.

https://phabricator.endlessm.com/T31182

### CODE REVIEW

**Comment by erikos** [comment]
This is blocked on landing https://github.com/endlessm/eos-build/pull/1212 and have a successful build result.

**Comment by wjt** [comment]
retest this please, oh wise robot

**Comment by wjt** [comment]
retest this please

@erikos you got PR 666! :smiling_imp: :metal:

**Comment by wjt** [comment]
Not sure what's happening. I guess there is latency between the package building on OBS and the apt repo updating.

File: eos-core-depends [hunk 1]
```diff
@@ -59,6 +59,7 @@ eos-hack-extension
 eos-installer
 eos-kalite-system-helper
 eos-kalite-tools
+eos-kolibri
```
Comment (wjt) [inline eos-core-depends:62]: The binary package is called:

```suggestion
eos-kolibri-system-helper
```

**Pull request merged**